apps/design/backend/test/helpers.ts

```diff
@@ -91,13 +77,49 @@ export class MockGoogleCloudTextToSpeechClient
       [{ audioContent: string | Uint8Array }, undefined, undefined]
     > =>
       Promise.resolve([
-        { audioContent: `${input.input.text} (audio)` },
+        { audioContent: mockCloudSynthesizedSpeech(input.input.text) },
         undefined,
         undefined,
       ])
   );
 }
 
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export function testSetupHelpers() {
```

Comment:
Moved this function from above since it now needs access to the `MockGoogleCloudTextToSpeechClient` and `MockGoogleCloudTranslationClient` also defined in this file